src/main/config/config.c

```diff
@@ -470,7 +470,7 @@ static void resetConf(void)
     masterConfig.escAndServoConfig.maxthrottle = 2000;
     masterConfig.motor_pwm_rate = 32000;
     masterConfig.looptime = 2000;
-//    currentProfile->pidController = 3;
+    currentProfile->pidController = 3;
```

Comment:
This will only be default for the ALIENWIIF1 and ALIENWIIF3 target. Everything else is unchanged. Please see also updated documentation.